pkg/cli/kubernetes/kubernetes.go

```diff
@@ -49,6 +50,7 @@ func init() {
 	_ = apiextv1.AddToScheme(Scheme)
 	_ = clientgoscheme.AddToScheme(Scheme)
 	_ = contourv1.AddToScheme(Scheme)
+	_ = radappiov1alpha3.AddToScheme(Scheme)
```

Comment:
Added so we can interact with the recipe CRD type in our code. There's no real drawback to this.